.github/workflows/regression.yml

```diff
@@ -43,7 +43,7 @@ jobs:
       run: python -m pip install --upgrade pip
     - name: Install dependencies
       run: |
-        brew install icarus-verilog
+        brew install icarus-verilog wget coreutils
```

Comment:
Added so that HardFloat can be installed with wget on macOS in the CI infrastructure